notebooks/experimental/prediction_tensorflow_model.ipynb

```diff
@@ -8,6 +8,14 @@
     "# Unfolded SDK Machine Learning Demo"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "05c04f6f-6bdb-4cf6-8a4e-64639c262a74",
+   "metadata": {},
+   "source": [
+    "The purpose of the model is to predict the median_house_value in California and then clustering it by the predicted prices."
+   ]
+  },
```

Comment:
I think it'd be good to clear all the cells output and save before merging:

![image](https://user-images.githubusercontent.com/351828/133582157-55383125-4794-48bb-ad53-c5d8b4724e6a.png)

Reply:
If the Unfolded maps worked from a saved html bundle, it seems it would be nice to save outputs in the notebook because a user could browse the output without needing to run the notebook. But we can explore that in a future iteration.